pillar.example

```diff
@@ -1,18 +1,18 @@
 postgres:
-  # Set True to configure upstream postgresql.org repository for YUM or APT
+  # Set True to configure upstream postgresql.org repository for YUM/APT/ZYPP
   use_upstream_repo: False
   # Version to install from upstream repository
-  version: '9.3'
+  version: '9.6'
 
   # These are Debian/Ubuntu specific package names
-  pkg: 'postgresql-9.3'
-  pkg_client: 'postgresql-client-9.3'
+  pkg: 'postgresql-9.6'
+  pkg_client: 'postgresql-client-9.6'
```

Comment:
As the comment above said, these values are specific to Ubuntu, 14.04 to be precise.
This is because the `pillar.example` is used in `test-kitchen` within the formula.
That's kind of legacy, but still serves as smoke test for regressions.

Since now we have good support for package names mapping for various distros, I think it could be safely commented out.

Reply:
Can we go further and remove all package names from pillar.example?

Executing the formula with pillars `use_upstream_repo: True` with `pkg_extras: ['postgresql-contrib', 'postgresql-plpython']` corrupts the postgres installation for example. They are not needed imho.

Reply:
And remove`service` pillar too.    And pkgs_extra example pillar implies `postgres-contrib` is optional "extra package", but the `postgres.prepare_cluster.command` fails (cannot find initdb) if package is not installed, implying its needed.

Reply:
You're right, the package and service definitions in Pillar are mostly unneeded.
But they could be useful when trying to install something on a latest-greatest and still unsupported distro.
I think it would be better to add a comment that would mention they are unnecessary in most cases.

Reply:
I'll think about this, thanks.

Reply:
Sure, just for now I think we should comment them out to make `test-kitchen` work, but add a proper description for these parameter later.
Or use generic ones like `postgresql` and `postgresql-client`. It will make transition to newer distro version much smoother as well.